js/widgets/rhythmruler.js

```diff
@@ -2671,9 +2671,14 @@ function RhythmRuler() {
 
         console.debug(rhythmarray);
 
-        if(rhythmarray != undefined) {
-            this._dissectHistory = rhythmarray;
+        var firsttime = true;
+        if(firsttime) {
```

Comment:
If you take the approach I was suggesting (a la Start) you won't have the first-time issue as the "history" will only be loaded when the block itself is loaded.